spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/HBaseConnectionCache.scala

```diff
@@ -90,14 +90,14 @@ private[spark] object HBaseConnectionCache extends Logging {
   def performHousekeeping(forceClean: Boolean) = {
     val tsNow: Long = System.currentTimeMillis()
     connectionMap.synchronized {
-      connectionMap.foreach {
-        x => {
-          if(x._2.refCount < 0) {
+      connectionMap.foreach { x =>
+        {
```

Comment:
shouldn't be 
```
 x => {
```
why there is a new line here...

Reply:
Could set `newlines.beforeCurlyLambdaParams = true` to prevent this. Let me add this in our scalafmt. Let me if you want this as an addendum fix for pr-119 or part of this PR itself. For now added as 2nd commit along with spotless re-run with change.